### DISCUSSION TITLE
Trying to configure the template path - clicking the textbox shows nothing

### DISCUSSION THREAD

**Discussion by stecydube** [chhoumann/quickadd#144] [Q&A] [answered]
I've installed QuickAdd and added a template as a choice.
Going into configure for this choice presents me all the options I can set - good ;)
However, clicking inside the "template path" textbox does not pop up the file tree...
So I manually entered the path and now it complains, when I try to run the choice, that it can't find it.

Am I missing something in the config?

**Comment by stecydube** [comment]
It seems I need to have the core template module enabled.
Is that a requisite for using this QuickAdd plugin?

**Reply by stecydube** [reply]
# Proof I'm not crazy 😜 (merely to convince myself 😂)


![Bug](https://user-images.githubusercontent.com/5737502/153525595-ac4db2ce-3653-470b-b984-6a6694c285c9.gif)

**Comment by chhoumann** [comment, marked as answer]
Coming back to this one very late - sorry about that.

You weren't crazy - back then QuickAdd sourced its template suggestions from the core Templates plugin's folder setting, which is exactly what your gif shows. That dependency is long gone: QuickAdd now has its own **Template folder paths** setting (QuickAdd settings -> Templates & Properties). Leave the list empty and the template path field suggests every template file in the vault; add folders to scope it down. No core Templates or Templater required.

Docs: https://quickadd.obsidian.guide/docs/Settings

Closing as outdated since the behavior you hit no longer exists. If anything similar comes up on a current version, a fresh issue is very welcome.
